test/resumable-upload.ts

```diff
@@ -53,6 +53,7 @@ const CHUNK_SIZE_MULTIPLE = 2 ** 18;
 const queryPath = '/?userProject=user-project-id';
 const X_GOOG_API_HEADER_REGEX =
   /^gl-node\/(?<nodeVersion>[^W]+) gccl\/(?<gccl>[^W]+) gccl-invocation-id\/(?<gcclInvocationId>[^W]+) gccl-gcs-cmd\/(?<gcclGcsCmd>[^W]+)$/;
+const USER_AGENT_REGEX = /^gcloud-node-storage\/(?<libVersion>[^W]+)$/;
```

Comment:
I'm confused, i think this should just be `gcloud-node/<libVersion>`;

Reply:
So I just moved around the existing code for setting `user-agent` (https://github.com/googleapis/nodejs-storage/blob/f20ef3cb7bf8cbdda988e792edc7abd9a1b516f1/src/nodejs-common/util.ts#L1010). Looks like it has always been set to `gcloud-node-storage` which seems to line up with the examples from the Headers document.

Reply:
TIL that this is different in Node.js thanks!